projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/components/ai-assistant-plugin-sidebar/index.tsx

```diff
@@ -109,12 +113,18 @@ const JetpackAndSettingsContent = ( {
 					<Upgrade placement={ placement } type={ upgradeType } upgradeUrl={ checkoutUrl } />
 				</PanelRow>
 			) }
-			{ isUsagePanelAvailable && (
+			{ isUsagePanelAvailable && showUsagePanel && (
 				<PanelRow className="jetpack-ai-sidebar__feature-section">
 					<UsagePanel placement={ placement } />
 				</PanelRow>
 			) }
 
+			{ showQuotaExceeded && (
```

Comment:
We may need to move it up, and find a way to change it's style to be color-less, since the designs are prescribing a message at the top of the sidebar, without the notice style: p1HpG7-tUE-p2. I was going to work on it today in the context of the UI changes related to fair usage. Feel free to keep it here while I don't get the PR out, our move it up and then I change the style to match designs.